### PR TITLE
Feature/#4 increase max wal size

### LINKED ISSUE #4: increase max_wal_size
To further stabilize the update process, `max_wal_size` should be incresed. See warning message below:

```
postgis_1      | LOG:  checkpoints are occurring too frequently (2 seconds apart)
postgis_1      | HINT:  Consider increasing the configuration parameter "max_wal_size".
```

### PR DESCRIPTION
**Pull request by hixi**
Increase max_wal_size to avoid running cleanup all too often.

Reviewed by:
- [x] @das-g

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,3 +87,6 @@ The ones we are using are:
   -N MAX-CONNECT     maximum number of allowed connections
   -S WORK-MEM        set amount of memory for sorts (in kB)
 ```
+
+Additionally, to get a better update-performance, we use
+the `max_wal_size=5GB` as run-time options.
```
Comment (das-g) [inline README.md:92]: Why not just list this with the options above? Only because this option isn't documented by `--help`?

File: README.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ docker-compose up -d postgis
 docker-compose run --rm \
   -e osmupdate_extra_params="--base-url=download.geofabrik.de/europe/switzerland-updates/" \
   -e osm_planet_mirror="http://download.geofabrik.de/" \
-  -e osm_planet_path_relative_to_mirror="europe/switzerland-160321.osm.pbf" \
+  -e osm_planet_path_relative_to_mirror="europe/switzerland-160301.osm.pbf" \
```
Comment (das-g) [inline README.md:61]: Why are these files ["more likely to be available"](https://github.com/geometalab/osm-world-db/pull/5/commits/b10cf26be21a7d47bdfa7bfac7981d1eafb86788)?

Comment (hixi) [inline README.md:61]: Because geofabrik keeps the files from the first of month around longer than the rest.

File: README.md (hunk 1 above)
Comment (hixi) [inline README.md:92]: Because it is not an option in the same sense as the ones above. Not sure if it would lead to confusion, I'm happy to add it above.

**Pull request merged**